thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/events/InformedQueryUtil.java

```diff
@@ -0,0 +1,174 @@
+package com.linkedin.thirdeye.anomaly.events;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+import java.util.List;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class InformedQueryUtil {
+  private final static Logger LOG = LoggerFactory.getLogger(InformedQueryUtil.class);
+  static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  // 1000ms: the smallest time unit is 1 second. 1ms: smallest unit is 1 ms.
+  // The value depends on the backend database (e.g., MySQL < 5.7 is 1000ms).
+  public static int TIME_PRECISION_IN_MILLIS = 1000;
+
+  private final static String INFORMED_QUERY = "?topics=%s";
+
+  private String queryPrefix;
+
+  public InformedQueryUtil(String informedUrl) {
+    this.queryPrefix = informedUrl + INFORMED_QUERY;
+  }
+
+  public InFormedPosts retrieveInformedEvents(String informedEventName, DateTime startTimeInclusive,
+      DateTime endTimeExclusive) {
+    long startMillis = startTimeInclusive.getMillis() / TIME_PRECISION_IN_MILLIS;
+    long endMillis = endTimeExclusive.getMillis() / TIME_PRECISION_IN_MILLIS;
+    String fetchUrl =
+        new StringBuilder(String.format(queryPrefix, informedEventName)).append("&start=")
+            .append(startMillis).append("&end=").append(endMillis).toString();
```

Comment:
Not sure if it is. But it would matter only when we start storing these event to our local db, at that point we would not want to store duplicates.